modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/FrontendDataSet.tsx

```diff
@@ -1149,9 +1151,12 @@ const FrontendDataSet = ({
 					)}
 
 					<div
-						className={`data-set-wrapper visualization-mode-${activeView.contentRenderer}`}
-						data-testid={`visualization-mode-${activeView.name}`}
-						ref={wrapperRef}
```

Comment:
afaik, this ref is needed, not just in the PR about file drop, but also [here](https://github.com/liferay-frontend/liferay-portal/pull/4968/files#diff-9e4260734ad825e1103118269eb3db433c4359f2b42d09ec31e76628e33c57bfL488)